remedy.lic

```diff
@@ -187,17 +188,19 @@ class Remedy
         # in cases where it won't get more herbs, it takes a longer route to get herbs from other bags (not @bag nor @herb_container)
         @count = Integer(bput("count my #{@herb1}", '\d+'))
         while @count < 25
-          bput("put my #{@herb1} in my #{@container}", 'You put', 'already has something in it.')
+          bput("put my #{@herb1} in my #{@container}", 'You put', 'already has something in it.', 'you count off and place only that many inside')
+          fput("stow my #{@herb1}") #in case there are extra
           bput("get my #{@container}", 'You pick up', 'You get')
+          # stow then get, to prevent getting herbs from container (i.e. mortar)
           stow_item(@container)
           bput("get my #{@herb1}", 'You get', 'What were you referring to?', 'You are already holding that.') # using bput instead of get_item to keep script running when it cannot get the herb
-          fput("get my #{@container}")
+          get_item(@container)
           bput("lower #{@container} ground", 'You lower')
           bput("get #{@herb1} from #{@container}", 'You get', 'What were you referring to?')
           if bput("combine my #{@herb1}", 'You combine', 'too large to add', 'You must be holding both') == 'You must be holding both'
-            @count = 25
+            @count = 25 # 25 used to break loops
```

Comment:
💡 If this is really just used to break the loop, then `break` is the better way to do this

Reply:
There are 2 separate loops using count, to handle some inventory issues. I think you have a point, so i looked at it again. At that point, it's meant to break both loops, because there are no more herbs to get.

I think i can optimize that section more, and replace that line with a break (i have to make a few changes for that to work). I've only programmed for personal projects, never professionally, so I still have plenty to learn.